docs/commands/addons.md

```diff
@@ -10,8 +10,7 @@ For more information on add-ons see our [Netlify partner add-ons docs](https://d
 
 ## About
 
-<!-- AUTO-GENERATED-CONTENT:START (GENERATE_COMMANDS_DOCS) -->
-(Beta) Manage Netlify Add-ons
+> ⚠️ Addons has been deprecated and will become unavailable in a future update.
```

Comment:
Wondering if we can use the same language as our release phases (`end of service` instead of `unavailable`) and link to the page: https://docs.netlify.com/platform/release-phases/#deprecated

I'm including a slight grammar edit here also:

```suggestion
> ⚠️ The addons command is [deprecated](https://docs.netlify.com/platform/release-phases/#deprecated) and will be end of service in a future update.
```

Though I see we used a similar message for large media, so I'm ok with just adding a link to the page but leaving the rest the same

```suggestion
> ⚠️ The addons command is [deprecated](https://docs.netlify.com/platform/release-phases/#deprecated) and will become unavailable in a future update.
```